service/src/main/java/bio/terra/externalcreds/visaComparators/RASv1_1.java

```diff
@@ -69,6 +71,8 @@ public boolean visaTypeSupported(GA4GHVisa visa) {
     @JsonProperty("consent_group")
     String getConsentGroup();
 
+    String getRole();
```

Comment:
I added role as a required field to detect when it changes. I don't think it really matters but seems better to send a notification and let downstream systems figure it out.